src/qubex/experiment/experiment.py

```
@@ -3992,10 +4005,12 @@ def measure_phase_shift_by_transmission_line(
         self,
         target: str,
         *,
-        frequency_range: ArrayLike | None = None,
+        frequency_range: ArrayLike = np.arange(10.05, 10.1, 0.002),
```

Ruff

Do not perform function call `np.arange` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)